ons_alpha/bulletins/blocks.py

```diff
@@ -25,6 +26,7 @@ class BulletinStoryBlock(StreamBlock):
     embed = EmbedBlock()
     image = ImageChooserBlock()
     related_links = RelatedLinksBlock(RelatedContentBlock())
+    chart = SnippetChooserBlock("core.Chart", template="templates/components/streamfield/chart.html", icon="table")
```

Comment:
non-blocking: this is where https://github.com/wagtail-nest/wagtail-font-awesome-svg would come in handy

https://fontawesome.com/search?q=chart&o=r&m=free